dex-it-common/src/main/scala/com/wavesplatform/dex/it/api/RawHttpChecks.scala

```diff
@@ -0,0 +1,34 @@
+package com.wavesplatform.dex.it.api
+
+import com.softwaremill.sttp.StatusCodes
+import org.scalatest.matchers.should.Matchers
+import play.api.libs.json.Json
+
+trait RawHttpChecks extends Matchers {
+
+  protected def validate200Json[ErrorT, EntityT](r: EnrichedResponse[ErrorT, EntityT]): EntityT = {
+    r.response.code should be(StatusCodes.Ok)
+    r.response.headers should contain("Content-Type", "application/json")
+    r.unsafeGet
+  }
+
+  protected def validate301Redirect[ErrorT, EntityT](r: EnrichedResponse[ErrorT, EntityT]): Unit =
+    r.response.code should be(StatusCodes.MovedPermanently)
+
+  protected def validateMatcherError[ErrorT, EntityT](r: EnrichedResponse[ErrorT, EntityT], c: Int, e: Int, m: String): Unit = {
+    r.response.code should be(c)
+    r.response.headers should contain("Content-Type", "application/json")
+    r.response.body should be leftSideValue
+
+    val b = Json.parse(r.response.body.left.get)
+    (b \ "message").as[String] should be(m)
+    (b \ "error").as[Int] should be(e)
```

Comment:
1. Let's get better names for c, e and m
2. You could do the same thing with less efforts:
    ```scala
    protected def validateMatcherError[EntityT](r: EnrichedResponse[MatcherError, EntityT],... = {
      // ...
      val b = r.tryGet.left.get
    ```